.github/workflows/publish.yml

```
@@ -0,0 +1,37 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ${{ matrix.os }}
    strategy:
      fail-fast: false
      matrix:
        os: [ubuntu-latest, windows-latest, macos-latest]
```

i tested on ubuntu (the `cibuildwheel` used docker images).
unsure of the other two OS's.

---

@erikbern Is this package meant to work on Windows and macOS?

---

I can confirm that it does build on macOS, and afaik people using a project that depends on this have encountered "lack of C++ compiler" errors on windows - seems like someone even published a windows-specific wheel [here](https://pypi.org/project/annoy-binary/#files)